frontend/src/features/help/components/HelpModal.scss

```diff
@@ -1,8 +0,0 @@
-.help-modal {
-  max-width: 700px;
```

Comment:
getting rid of this may have caused your display issue.

Reply:
Getting rid of it solved my display issue.